card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
```

Comment:
Remove commented code

Reply:
**Resolved:** Removed the commented code